components/dashboard/src/settings/Preferences.tsx

```diff
@@ -153,6 +161,12 @@ export default function Preferences() {
                     </div>
                 </SelectableCard>
             </div>
+            <h3 className="mt-12">Dotfiles <PillLabel type="warn" className="font-semibold mt-2 py-0.5 px-2 self-center">Beta</PillLabel></h3>
+            <p className="text-base text-gray-500 dark:text-gray-400">Customise every workspace using dotfiles. Add a repo below which gets cloned and installed during workspace startup.</p>
```

Comment:
**nitpick**: So far, we've generally opted for US English across the dashboard. 🇺🇸 

**question**: Also, what do you think of moving the input help text closer to the input instead of using the section subtitle?

```suggestion
            <p className="text-base text-gray-500 dark:text-gray-400">Customize workspaces using dotfiles.</p>
```

**suggestion**: A link to learn more about how this feature works (conventions, installation, etc) to the docs (https://github.com/gitpod-io/website/issues/1407) once the docs are in place could be also nice, as @jldec mentioned in https://github.com/gitpod-io/gitpod/pull/7337#issuecomment-1002740006.